causal_testing/testing/estimators.py

```diff
@@ -282,17 +282,17 @@ class LinearRegressionEstimator(Estimator):
 
     def __init__(
         self,
-        treatment: tuple,
-        treatment_values: float,
-        control_values: float,
-        adjustment_set: list[float],
+        treatment: tuple,    
+        treatment_value: float,
+        control_value: float,
+        adjustment_set: set,
```

Comment:
This is another change that has had impacts elsewhere. Seems like it enables some quite neat code, but I guess we've lost control of the types that go in the set. `set[float]`?

Reply:
The change in typing for `adjustment_set` was to make it consistent with the typing of `adjustment_set` in the other estimator classes.